IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcDoorType/Documentation.md

```diff
@@ -1,34 +1,48 @@
+* 'Framing' - to indicate that the material constituent applies to to the door framing, if not provided, the 'Lining' material information applied to frams as well
 * 'Glazing' - to indicate that the material constituent applies to to the glazing as well
 
-  
-If the fall back single _IfcMaterial_ is referenced, it applies to the lining and framing of the door.  
-  
-**Geometry Use Definitions:**  
-The _IfcDoorType_ may define the common shape of door occurrences. The common shape can be defined by  
-* applying shape parameters defined within the associated _IfcDoorLiningProperties_ and _IfcDoorPanelProperties_ applied to the ''Profile'' geometric representation. It is only applicable if the _IfcDoorType_ has only occurrences of type _IfcDoorStandardCase_ (See geometric use definition of _IfcDoorStandardCase_ for further information).
-* applying the _RepresentationMaps_ attribute to refer to a list of _IfcRepresentationMap_''s, that allow for multiple geometric representations (e.g. with _IfcShapeRepresentation_''s having an _RepresentationIdentifier_ ''Box'', ''Profile'', ''FootPrint'', or ''Body'')
-
-  
-NOTE The product shape representations are defined as _RepresentationMaps_ (attribute of the supertype _IfcTypeProduct_), which get assigned by an element occurrence instance through the _IfcShapeRepresentation.Item[n]_ being an _IfcMappedItem_. See _IfcTypeProduct_ for further information.   
-NOTE The values of attributes _RepresentationIdentifier_ and _RepresentationType_ of _IfcShapeRepresentation_ are restricted in the same way as those for _IfcDoor_ and _IfcDoorStandardCase_
+If the fall back single _IfcMaterial_ is referenced, it applies to the lining and framing of the door.
+
+{ .use-head}
+Geometry Use Definitions:
+
+The _IfcDoorType_ may define the common shape of door occurrences. The common shape can be defined by
+
+* applying shape parameters defined within the associated _IfcDoorLiningProperties_ and _IfcDoorPanelProperties_ applied to the 'Profile' geometric representation. It is only applicable if the _IfcDoorType_ has only occurrences of type _IfcDoorStandardCase_ (See geometric use definition of _IfcDoorStandardCase_ for further information).
+* applying the _RepresentationMaps_ attribute to refer to a list of _IfcRepresentationMap_'s, that allow for multiple geometric representations (e.g. with _IfcShapeRepresentation_'s having an _RepresentationIdentifier_ 'Box', 'Profile', 'FootPrint', or 'Body') 
+>> NOTE&nbsp; The product shape representations are defined as _RepresentationMaps_ (attribute of the supertype _IfcTypeProduct_), which get assigned by an element occurrence instance through the _IfcShapeRepresentation.Item[n]_ being an _IfcMappedItem_. See _IfcTypeProduct_ for further information. 
+>> NOTE&nbsp; The values of attributes _RepresentationIdentifier_ and _RepresentationType_ of _IfcShapeRepresentation_ are restricted in the same way as those for _IfcDoor_ and _IfcDoorStandardCase_
```

Comment:
This should be deleted also but cannot propose it because some of the lines have already been deleted.

Reply:
I deleted this in latest commit.